mentors/dart/c.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "Corentin-Leffy",
+    "name": "Corentin Leffy",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I'm a passionate and professional software developer from France, interested about Android, Kotlin, Dart, TDD, ...",
```

Comment:
I didn't notice this before.  Remove the trailing comma, last lines don't need them.